apps/supernova/src/components/silences/RecreateSilence.jsx

```diff
@@ -82,6 +83,9 @@ const RecreateSilence = (props) => {
     setError(null)
     setSuccess(null)
     setShowValidation({})
+    if (authData?.parsed?.fullName === "anonymous") {
```

Comment:
this looks like too weak... I would suggest to use at least a constant which is used in the setter and here. But the best would be to set a flag in the AuthData object which defines that there is no authentication a sets default attributes. what do you think?